fbw-a32nx/src/systems/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -754,6 +776,22 @@ class MetricAltIndicator extends DisplayComponent<{ bus: EventBus }> {
         sub.on('realTime').handle(this.updateState.bind(this));
     }
 
+    private updateAltitudeColor() {
+        if (this.state.altitudeColor === TargetAltitudeColor.White) {
+            this.metricAltTargetText.instance.classList.remove('Cyan');
+            this.metricAltTargetText.instance.classList.remove('Magenta');
+            this.metricAltTargetText.instance.classList.add('White');
+        } else if (this.state.altitudeColor === TargetAltitudeColor.Magenta) {
+            this.metricAltTargetText.instance.classList.remove('Cyan');
+            this.metricAltTargetText.instance.classList.remove('White');
+            this.metricAltTargetText.instance.classList.add('Magenta');
+        } else {
+            this.metricAltTargetText.instance.classList.remove('Magenta');
+            this.metricAltTargetText.instance.classList.remove('White');
+            this.metricAltTargetText.instance.classList.add('Cyan');
+        }
+    }
```

Comment:
Same for SelectedAltIndicator too
```suggestion
    private updateAltitudeColor() {
         this.metricAltTargetText.instance.classList.toggle('Cyan', this.state.altitudeColor === TargetAltitudeColor.Cyan);
         this.metricAltTargetText.instance.classList.toggle('Magenta', this.state.altitudeColor === TargetAltitudeColor.Magenta);
         this.metricAltTargetText.instance.classList.toggle('White'. this.state.altitudeColor === TargetAltitudeColor.White);
    }
```

Reply:
Ah, I knew there had to be a better way to do this 😄. Thanks for the suggestion.